termlink/__main__.py

```diff
@@ -158,6 +158,13 @@
     help="resource identifier for files"
 )
 
+parser_gsea.add_argument(
```

Comment:
Cool. I've been working on abstracting the IO to the top-level command. I'll add this to the story.